decentraland/unity-explorer
# fix: map filters POI tooltip position

## What does this PR change?

This PR was created to fix the issue #956 . As the nav bar will always be on top of every other UI I decided to move the tooltip to the right side of the info button instead of keeping it at the top.
<img width="269" alt="Screenshot 2024-05-07 at 10 07 39" src="https://github.com/decentraland/unity-explorer/assets/51088292/0c5d1064-5911-4124-ac2b-ad2dddda3c81">


## How to test the changes?

1. Launch the explorer.
2. Open the nav map pressing **M** or the mini map.
3. Click the filters dropdown and then the info button.
4. Check the tooltip is not being overlap by the nav bar anymore.
